config.yaml

```diff
@@ -13,6 +13,10 @@ receivers:
     transport: udp
 
 processors:
+  resourcedetection/ec2:
+    detectors: [ env ]
+    timeout: 2s
+    override: false
```

Comment:
can we remove it since it is not used by the pipeline?

Reply:
Removed.